spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -2165,17 +2151,6 @@ def interrupt_kernel(self):
             self.sig_switch_to_plugin_requested.emit()
             client.stop_button_click_handler()
 
-    def connect_external_spyder_kernel(self, shellwidget):
```

Comment:
The shellwidget automatically updates when receiving confirmation that it is a spyder kernel, so no specific action needs to be taken.